src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -93,12 +93,19 @@ static void compoundObjectResponse(orion::CompoundValueNode* cvP, const BSONElem
 *
 * mongoConnect -
 */
-bool mongoConnect(const char* host, const char* db, const char* rplSet, const char* username, const char* passwd, bool _multitenant) {
+bool mongoConnect(const char* host,
+                  const char* db,
+                  const char* rplSet,
+                  const char* username,
+                  const char* passwd,
+                  bool        _multitenant,
+                  double      timeout)
+{
 
     std::string err;
     multitenant = _multitenant;
 
-    LM_T(LmtBug, ("dbName: '%s'", db));
+    LM_T(LmtBug, ("dbName: '%s', rplSet: '%s', timeout: %f", db, rplSet, timeout));
```

Comment:
Fixed in bd187d2. Some LM_T with LmtBug has been removed in the process (I think they were stale LM_T traces).